garage/envs/mujoco/randomization/randomized_env.py

```diff
@@ -73,9 +73,5 @@ def action_space(self):
     def observation_space(self):
         return self._wrapped_env.observation_space
 
-    @property
-    def horizon(self):
```

Comment:
Why is horizon completely removed instead of renamed/replaced?

Reply:
I see, it can be replaced with env.spec.max_episode_steps. Although if we're adding a lot of wrappers to the env, for example, two wrappers, we would need to call it like env.env.env.spec.max_espisode_steps, right?

Reply:
Yes, but that's why I also added `max_episode_steps` functions to all of our env wrappers.

Reply:
So we wouldn't need several levels of env.env.env

Reply:
Okay, but randomize is also used as an env wrapper and you didn't add max_episodes_steps to it.

Reply:
Good catch, I'll add it now.